documentation/API-onboarding.md

```diff
@@ -0,0 +1,75 @@
+
+## Submission of an API proposal
+
+The pipeline is triggered with the submission of an API proposal. This API proposal can represent a new API or a new API family.
+
+API proposals can be submitted by everybody, e.g. GSMA OPAG, external or CAMARA member companies, and existing CAMARA Sub Projects or Working Groups (if these realize that the scope has to be increased).
+
+To proceed with the submission, the API proposal owner shall follow these steps:
+
+- Fill in the template available [here](https://github.com/camaraproject/WorkingGroups/blob/main/Commonalities/documentation/API-proposal-template.md) and save it with the following name: "APIproposal\_\<APIname\>\_\<owner\>. md" locally.
+- Create a new issue in the API Backlog Working Group repository, labeled with "API Backlog".
+- Upload the filled-in template to [GitHub repository folder for API proposals](https://github.com/camaraproject/WorkingGroups/tree/main/APIBacklog/documentation/SupportingDocuments/API%20proposals) via pull request. This pull request shall be associated to the issue created in the previous step.
+
+## Evaluation of the API proposal
+
+Upon submission, the "API Backlog" Working Group (WG) validates the API proposal. In particular:
+
+- The API Backlog WG coordinator checks that the template is duly filled in. Otherwise, the API proposal owner will be requested to provide missing information.
+- After this sanity check, each WG participant declares his/her support (which is also documented in the template).
+
+Finally the pull request is merged into the main branch and the API proposal template is sent to the Steering Committee of CAMARA at least one week prior to the Steering Committee meeting where the API proposal shall be voted upon.
+
+This whole step shall be done within 2 regular meetings of the API Backlog WG. In parallel, the API proposal is formally registered in CAMARA by adding it to the [CAMARA API overview list](https://github.com/camaraproject/WorkingGroups/blob/main/APIBacklog/documentation/APIBacklog.md) (API Backlog WG creates a new line in the table and fills in API family name, API family owner, API family proposal registration date, the link to the application template and the supporters for it).
+
+## API proposal Steering Committee decision
+
+Upon receiving the API proposal and notification from the API Backlog WG, the Steering Committee studies the proposal and votes it at the Steering Committee meeting.
+
+- No-Go! Objecting companies shall provide justifications why. The API proposal is rejected and will not be included in any API Sub Project.
+- Go! In this case, the Steering Committee shall specify whether the API proposal is to be hosted by a new or existing Sub Project.
+
+Main tenets for Steering Committee voting:
+
+- Tenet #1: Open-source projects are contribution-driven. There should be no guidelines to prevent partners to contribute. If a company would like to work on a certain API proposal they should be welcome to do it.
```

Comment:
I assume in most cases No Gos would occur in order to avoid API duplication. So should there be a "No Go with check" state to ask the submitter to see if an already available, strongly related API could be used / adapted instead.

Reply:
@tanjadegroot : I'll check the wording and change it more openly, so that the tsc can decide to open a new Sub Project, to put the API proposal in an existing Sub Project (even with modifications), or to refuse it (e.g. because it's in conflict with regulations or guidelines).

Reply:
perfect ! thanks Markus